perfkitbenchmarker/scripts/database_scripts/file_to_plot.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
file_to_plot.py doesn't seem like a good name for this tool. This tool converts an output file which contains detail results of a mysql_service_benchmark run (i.e., sysbench run) to a gnuplot graph. So the name should reflect that.

plot_sysbench_results.py or something like that.